framework/generated/generated_vulkan_feature_util.cpp

```diff
@@ -63,23 +79,43 @@ void RemoveUnsupportedFeatures(VkPhysicalDevice physicalDevice, PFN_vkGetPhysica
                 GetPhysicalDeviceFeatures2(physicalDevice, &physicalDeviceFeatures2);
                 if ((currentNext->storageBuffer16BitAccess == VK_TRUE) && (query.storageBuffer16BitAccess == VK_FALSE))
                 {
-                    GFXRECON_LOG_WARNING("Feature storageBuffer16BitAccess, which is not supported by the replay device, will not be enabled");
-                    const_cast<VkPhysicalDevice16BitStorageFeatures*>(currentNext)->storageBuffer16BitAccess = VK_FALSE;
+                    GFXRECON_LOG_WARNING("Feature storageBuffer16BitAccess %s", warn_message);
+                    found_unsupported = true;
+
+                    if (remove_unsupported)
+                    {
+                        const_cast<VkPhysicalDevice16BitStorageFeatures*>(currentNext)->storageBuffer16BitAccess = VK_FALSE;
```

Comment:
If you want to save even more lines of code, you could do "... = !remove_unsupported;" here instead of the if clause :-)